synapse/models/infotech.py

```diff
@@ -144,6 +144,44 @@ def initCoreModule(self):
     def _onFormItDevStr(self, form, valu, props, mesg):
         props['it:dev:str:norm'] = valu.lower()
 
+    @modelrev('it', 201801041154)
+    def _revModl201801041154(self):
+
+        # mark changed nodes with a dark row...
+        dvalu = 'it:201801041154'
+        dprop = '_:dark:syn:modl:rev'
+
+        idens = []
+
+        # carve registry keys to being lower case normalized
+        with self.core.getCoreXact():
+
+            # bulk cut over all it:dev:regval:key props
+            rows = self.core.getRowsByProp('it:dev:regval:key')
+
+            adds = [(i, p, v.lower(), t) for (i, p, v, t) in rows]
+            darks = [(i[::-1], dprop, dvalu, t) for (i, p, v, t) in rows]
```

Comment:
I think the timestamp for these dark rows should be the current time the migration was executed; not the time from the existing rows.